builtin/builtin.mbti

```diff
@@ -209,10 +213,10 @@ impl Bytes {
   blit_from_string(Bytes, Int, String, Int, Int) -> Unit
   copy(Bytes) -> Bytes
   length(Bytes) -> Int
-  make(Int, Int) -> Bytes
+  make(Int, ~init : Byte = ..) -> Bytes
```

Comment:
make sure `moon test` is not broken, I have an impression that the test driver used this API cc @lijunchen

Reply:
The test driver depends on Buffer, which hides the implementation detail of using bytes. So `moon test` still working.